### ISSUE TITLE
Remove yellow border around active button

### ISSUE THREAD

**Issue by Brennan-Dan** [bitfocus/companion#1648]
**Describe the feature**
A clear and concise description of what the feature does
Remove Yellow border around active buttons, Like the ability to remove the top bar of the buttons.

**Usecases**
Add a couple of usecases for why this should be implemented and when/why to use it.

When using feedback control it isn't necessary to know if a button is active/latched when using feedback control. Would be a nice feature to be able to turn on/off as per the topbar setting.

**Comment by Mozoloa** [comment]
Please god yes, how many hours have I spent trying to work around the absence of this feature, my brain is a bowl of spaghettis

**Comment by Brennan-Dan** [comment]
In addition to disabling the yellow border like the top bar, the ability to turn off the play symbol that displays when a series is actions are playing out on the button. This is because the feedback state of the button is already showing me what is going on.

**Comment by Schrootpoot** [comment]
Would also love to see this. My usecase the border is completely unnecessary and makes it look a lot less clean than it could be.

**Comment by nqeldridge** [comment]
What consequence does this have on the button state itself?  The most annoying part of the yellow border in my experience is that when it is active, I have to press the button once to acknowledge the border and a second time to execute the actions associated with the button.  Is that functionality being resolved as part of this or does that need to be a separate feature request?

**Comment by Julusian** [comment]
> The most annoying part of the yellow border in my experience is that when it is active, I have to press the button once to acknowledge the border and a second time to execute the actions associated with the button.

No, this is not to change any behaviour, just how the buttons look visually.

So for you it will likely make that more confusing as there will be no visual indication that it needs to be pressed twice.

> I should add, this specifically is the case when using button presses as an action of another button.

That sounds like you have some quirks in your setup causing these buttons to be pushed but never released.

**Comment by Julusian** [comment]
This is possible in the betas now

<img width="812" height="857" alt="Image" src="https://github.com/user-attachments/assets/36b165c4-7d79-4b36-b445-1fc3fac413d4" />
